src/app/plugins-initializer/plugins-initializer.ts

```diff
@@ -0,0 +1,64 @@
+import { APP_INITIALIZER, Compiler, FactoryProvider, Injector } from '@angular/core';
+import { A1PluginDefinition } from './shared/a1-plugin-definition';
+import { A2PluginDefinition } from './shared/a2-plugin-definition';
+import { registerA1Plugins } from './register-a1-plugins';
+import { registerA2Plugins } from './register-a2-plugins';
+
+const OVERRIDE_PLUGINS = new Map<string, string>([['multitenancy', 'multitenancy/remoteEntry.js']]);
```

Comment:
Great, I think this is the right approach until we modified the BE! Maybe add a comment 
//TODO: modify in BE and remove override
just to make sure we don't forget